ONSdigital/dp-feedback-api
# Feature/component tests

### What

[Trello card](https://trello.com/c/0HEUJQ15/1140-feedback-api-add-component-test-coverage)

- Remove `hello` endpoint
- Implement component tests according to requirements
- Remove full handler unit tests for `feedback` endpoint

### How to review

- Make sure changes make sense and cover all the requirements
- Run component tests (`make test-component`) and check they pass

### Who can review

Anyone except me
